ios/brave-ios/Sources/BraveWallet/Crypto/Transactions/TransactionParser.swift

```diff
@@ -126,7 +126,8 @@ enum TransactionParser {
 
   static func parseTransaction(
     transaction: BraveWallet.TransactionInfo,
-    network: BraveWallet.NetworkInfo,
+    currentNetwork: BraveWallet.NetworkInfo,
```

Comment:
`currentNetwork` isn't needed anymore, can fetch it using the `chainId` & `coin` from the `TransactionInfo` model.